drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -6931,7 +6931,12 @@ void RasterizerStorageGLES3::_render_target_allocate(RenderTarget *rt) {
 
 			glGenTextures(1, &rt->exposure.color);
 			glBindTexture(GL_TEXTURE_2D, rt->exposure.color);
+#ifdef IPHONE_ENABLED
```

Comment:
you could probably add a check for this support at RasterizerStorageGLES3::initialize and then check a bool instead of harcoding it to IOS. I am sure there is an opengl extension that can be checked here.

Reply:
Will do!